### PR TITLE
Comet reward with support for multiple rewards per batch

### PR DESCRIPTION
**Pull request by DuarteMRAlves**

### CODE REVIEW

File: verl/workers/reward_manager/multiple.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import torch
+
+from collections import defaultdict
+from verl import DataProto
+
+class MultipleRewardManager:
```
Comment (CoderPat) [inline verl/workers/reward_manager/multiple.py:6]: So my understanding from reading this PR is that: (1) if RM / COMET rewards are active, they will be computed for all samples (2) they will be combined based on *weighting*. 
My question is then how do exactly *pick* a specific reward based on the sample? for reasoning we only want reward_function, for MT we only want COMET(+format). 
Maybe I missed this

**Review by CoderPat — APPROVED** [review]
Other than small clarifications, LGTM

File: verl/workers/reward_manager/multiple.py (hunk 1 above)
Comment (DuarteMRAlves) [reply verl/workers/reward_manager/multiple.py:6]: This is a bit of a trick, but each of them should put a reward of 0 for samples where reward model type is not theirs. Not ideal, but it is like a protocol all reward models must follow, in which they only score the samples for their type
